hermes/src/store/storage/local_storage.rs

```diff
@@ -0,0 +1,163 @@
+use {
+    super::{
+        AccumulatorState,
+        MessageState,
+        MessageStateFilter,
+        MessageStateKey,
+        RequestTime,
+        Storage,
+        StorageInstance,
+    },
+    crate::store::types::Slot,
+    anyhow::{
+        anyhow,
+        Result,
+    },
+    async_trait::async_trait,
+    dashmap::DashMap,
+    moka::sync::Cache,
+    pyth_oracle::MessageType,
+    pyth_sdk::PriceIdentifier,
+    std::{
+        collections::VecDeque,
+        sync::Arc,
+    },
+    strum::IntoEnumIterator,
+};
+
+#[derive(Clone)]
+pub struct LocalStorage {
+    message_cache:     Arc<DashMap<MessageStateKey, VecDeque<MessageState>>>,
+    accumulator_cache: Cache<Slot, AccumulatorState>,
+    cache_size:        u64,
+}
+
+impl LocalStorage {
+    pub fn new_instance(cache_size: u64) -> StorageInstance {
+        Box::new(Self {
+            message_cache: Arc::new(DashMap::new()),
+            accumulator_cache: Cache::builder()
+                .max_capacity(cache_size)
+                .time_to_live(std::time::Duration::from_secs(60 * 60 * 24))
```

Comment:
extract a constant? (ideally this should be configurable also?)

Reply:
also doesn't 1 day seem like a long time to keep things around in the cache? It could potentially be a lot of data?